app/assets/stylesheets/views/article.scss

```
@@ -150,6 +150,21 @@
     flex-direction: column;
   }
 
+  &.c-btn {
+    &:hover:enabled,
+    &:enabled,
+    &:active,
+    &[aria-pressed='true'] {
+      background-color: transparent;
+    }
+
+    @media (max-width: $breakpoint-m) and (hover: hover) {
```

```suggestion
    // We don't show tooltips on touch-only devices
    @media (max-width: $breakpoint-m) and (hover: hover) {
```

(just a non-blocking suggestion) - it might be nice to add a comment here to explain 🙂